content/topics/web-frontend/intro-to-vue/_index.md

```diff
@@ -1,3 +1,238 @@
 ---
 title: Intro to Vue
+ready: True
 ---
+
+##### what is ####
+
+[Vue.js](https://vuejs.org/v2/guide/index.html) is an [Model View viewmodel](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel) [javascript framework](https://en.wikipedia.org/wiki/JavaScript_framework) that has different optional tools for building user interfaces and [single page applications](https://en.wikipedia.org/wiki/Single-page_application). Vue.js allow you to extend basic HTML with html attributes called [directives](https://012.vuejs.org/guide/directives.html) which adds extra functionality to the html. This directives are provided by Vue.js by default or can be defined by user.
+
+_Vue.js Directives_
+
+In Vue.js a directive is a type of token/indentifier that tells vue.js code to do something to the [DOM/Documet Object Model](https://en.wikipedia.org/wiki/Document_Object_Model). 
+
+Directive simple syntax :
+
+{{< highlight go "linenos=table,linenostart=1" >}}
+
+ <html-element prefix-directive="expression"></html-element>
+
+{{< / highlight >}}
+
+_Usage example_
+
+{{< highlight go "linenos=table,linenostart=1" >}}
+
+ <h1 id='heading1' v-text='heading_1'></h1>
+
+{{< / highlight >}}
+This can also be written using double braces as placeholders for output data like :
+{{< highlight go "linenos=table,linenostart=1" >}}
+
+ <h1 id='heading1'>
```

Comment:
This is an example in the subsection and it is bigger that the sub-title, it looks a bit weird.